src/platform/user/profile/actions/index.js

```diff
@@ -46,8 +46,14 @@ export function initializeProfile() {
       await dispatch(refreshProfile());
       dispatch(updateLoggedInStatus(true));
     } catch (error) {
-      dispatch(updateLoggedInStatus(false));
-      teardownProfileSession();
+      /* If the fetch fails due to the browser cancelling the request due to a navigation event short circuit it to prevent terminating session */
+      if (error instanceof TypeError && error.message === 'Failed to fetch') {
+        // eslint-disable-next-line no-useless-return
+        return;
```

Comment:
Is this `return` necessary? It doesn't look like there is anything after the if statement that would happen if the `return` were removed.

Reply:
I'm going to refactor it to what you suggested and see if the same results.  Initially, any caught error tore down the profile but I will let you know what I find out with testing.

Reply:
Looks like it works as expected without the return statement; Pushing up modifications now.  Thanks for your suggested refactor 👍